woocommerce/woocommerce
# CYS - fix warning Tooltip

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #45486.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

A WordPress component triggers the error: https://github.com/WordPress/gutenberg/issues/43129

1. Make sure you have the WooCommerce Beta Tester plugin installed and activated
2. Under Tools > WCA Test Helper > Features, make sure customize-store is enabled
3. Under Tools > WCA Test Helper > Tools, click on Reset Customize Your Store
4. Head over to WooCommerce > Home > Customize your store
5. Click on "Start designing" and proceed to the Pattern Assembler
6. Click on "Choose your header"
7. Open the console and verify the following error is not visible:


<img width="1997" alt="Screenshot 2024-03-11 at 16 47 31" src="https://github.com/woocommerce/woocommerce/assets/15730971/7a1e42e9-d0b3-4111-9ede-b4cf02640a1d">



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

CYS - fix warning Tooltip

#### Comment 

</details>
